core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -74,29 +88,32 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
   import KafkaConsumerActor._
 
   val pollMsg = Poll(this)
-  def pollTimeout() = settings.pollTimeout
-  def pollInterval() = settings.pollInterval
+  var subscriptions = Map.empty[ActorRef, SubscriptionMessage]
+  var consumer: KafkaConsumer[K, V] = _
 
   var currentPollTask: Cancellable = _
-
-  var requests = Map.empty[ActorRef, RequestMessages]
-  var consumer: KafkaConsumer[K, V] = _
   var commitsInProgress = 0
   var wakeups = 0
   var stopInProgress = false
 
+  def pollTimeout() = settings.pollTimeout
```

Comment:
val is not necessarily better though, def is fine.
val would be worth checking if we still have space in the object without further extending its allocated size (due to alignment), and the def will likely be inlined anyway to that val so I would not worry about it unless proven to have impact